youtube_dl/extractor/hrfensehen.py

```diff
@@ -0,0 +1,74 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+import re
+
+from youtube_dl.utils import int_or_none
+from .common import InfoExtractor
+
+
+class HRFernsehenIE(InfoExtractor):
+    IE_NAME = 'hrfernsehen'
+    _VALID_URL = r'^https?://www\.(?:hr-fernsehen|hessenschau)\.de/.*,video-[0-9]{6}\.html'
+
+    _TESTS = [{
+        'url': 'https://www.hessenschau.de/tv-sendung/hessenschau-vom-26082020,video-130546.html',
+        'md5': '5c4e0ba94677c516a2f65a84110fc536',
+        'info_dict': {
+            'id': '130546',
+            'ext': 'mp4',
+            'description': 'Sturmtief Kirsten fegt über Hessen / Die Corona-Pandemie – eine Chronologie / '
+                           'Sterbehilfe: Die Lage in Hessen / Miss Hessen leitet zwei eigene Unternehmen / '
+                           'Pop-Up Museum zeigt Schwarze Unterhaltung und Black Music',
+            'title': 'hessenschau vom 26.08.2020'
+        }
+    }, {
+        'url': 'https://www.hessenschau.de/tv-sendung/hessenschau-vom-26082020,video-130546.html',
+        'only_matching': True
+    }]
+
+    _GEO_COUNTRIES = ['DE']
+
+    def _real_extract(self, url):
+        video_id = re.search(r'video-([0-9]+)', url).group(1)
+
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._html_search_meta(
+            ['og:title', 'twitter:title', 'name'], webpage)
+
+        description = self._html_search_meta(
+            ['description'], webpage)
+
+        media_streams = re.findall(r'{ \"url\":\".*\", \"verticalResolution\": [0-9]{3,5} }', webpage)
```

Comment:
I suggest it's better to look for `ardplayer` and then parse the content of the attribute `data-hr-mediaplayer-loader`. That way you get all and even more information but the description.

`data-hr-mediaplayer-loader="{
"type": "ondemand",
"position": "page",
"containerId": "8054",
"streamUrl":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_512x288-25p-500kbit.mp4",
"isAdaptiveStream": true,
"adaptiveStreamingUrl":"https://hr-vh.akamaihd.net/i/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_,320x180-12p-125kbit,480x270-25p-250kbit,512x288-25p-500kbit,640x360-25p-1000kbit,960x540-50p-1800kbit,1280x720-50p-5000kbit,1920x1080-50p-8000kbit,.mp4.csmil/master.m3u8",
"videoResolutionLevels": [
{ "url":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_320x180-12p-125kbit.mp4", "verticalResolution": 180 },
{ "url":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_480x270-25p-250kbit.mp4", "verticalResolution": 270 },
{ "url":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_512x288-25p-500kbit.mp4", "verticalResolution": 288 },
{ "url":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_640x360-25p-1000kbit.mp4", "verticalResolution": 360 },
{ "url":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_960x540-50p-1800kbit.mp4", "verticalResolution": 540 },
{ "url":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_1280x720-50p-5000kbit.mp4", "verticalResolution": 720 },
{ "url":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_1920x1080-50p-8000kbit.mp4", "verticalResolution": 1080 }
],
"subtitle":"https://hr-a.akamaihd.net/video/as/hessenschau/2020_08/hrLogo_200826200407_L385592_512x288-25p-500kbit.vtt",
"skinPath":"https://www.hessenschau.de/assets_2.107.1/vendor/ardplayer/ardplayer-hr.css",
"mediaMetadata":{"agf":{"title":"hr-fernsehen_hessenschau_hessenschau vom 26.08.2020_26.08.2020 19:30","livestream":"no","type":"content","airdate":"26.08.2020 19:30","assetid":"2a0dd5ac3c9efdd11ebc114578d316da","program":"hr-fernsehen","uurl":"as/hessenschau/2020_08/hrLogo_200826200407_L385592_512x288-25p-500kbit.mp4","nol_c2":"p2,N","nol_c5":"p5,hessenschau","nol_c7":"p7,as/hessenschau/2020_08/hrLogo_200826200407_L385592_512x288-25p-500kbit.mp4","nol_c8":"p8,1654","nol_c9":"p9,hessenschau_hessenschau vom 26.08.2020_26.08.2020 19:30","nol_c10":"p10,hr-fernsehen","nol_c12":"p12,Content","nol_c15":"p15,72B6616139694E87B4B0808C1D195B5E","nol_c16":"p16,hr-fernsehen_Information|Nachrichten","nol_c18":"p18,N","length":"1654","nol_c20":"p20,GFKLinkID"},"ati":{"pageName":"hessenschau_vom_26_08_2020_video","secondLevelId":"11","documentName":"video-130546","documentType":"hr-nt:video","documentDate":"202008262038","chapter":"","mediaLabel":"hessenschau_vom_26_08_2020_video","mediaTheme1":"hessenschau","mediaTheme2":""}},
"ardplayerUrl": "https://www.hessenschau.de/assets_2.107.1/js/vendor/ardplayerHr.js",
"smarttagUrl": "https://www.hessenschau.de/assets_2.107.1/js/vendor/smarttagProd.js",
"atiSite":551104,
"atiTrackingHost": "https://logs1406",
"agfAppId": "P726B1420-8C67-4A21-A691-9065C69349EA",
"baseUrl":"https://www.hessenschau.de/assets_2.107.1/vendor/ardplayer/"
,"previewImageUrl":{
"s": "https://www.hessenschau.de/tv-sendung/hs_ganz-1554~_t-1598465545029_v-16to9.jpg",
"m": "https://www.hessenschau.de/tv-sendung/hs_ganz-1554~_t-1598465545029_v-16to9.jpg",
"l": "https://www.hessenschau.de/tv-sendung/hs_ganz-1554~_t-1598465545029_v-16to9.jpg",
"xl": "https://www.hessenschau.de/tv-sendung/hs_ganz-1554~_t-1598465545029_v-16to9__medium.jpg"
}
}"`

Reply:
I didn't had subtitles on my radar. Can you have a look on my updated approach?

Reply:
> I didn't had subtitles on my radar. Can you have a look on my updated approach?

I had a quick look over it. Looks good so far :)
Tho I haven't tested the code myself. Maybe there is some `regex` alternative by youtube-dl but haven't dealt with that so far with youtube-dl. But don't worry about it. If it works, it works :D
Good luck that your PR will be merged anytime in the future 😟